codeforpdx/PASS
# Make Docker Config usable in development

# Description
The docker config merged earlier works for the opencommons deploys, but it has specific data that keeps it from being used in development. This PR reconfigures the config to be usable on any computer.

# Testing
1. Pull branch
2. Make sure docker is installed on your computer. Instructions [here](https://docs.docker.com/engine/install/)
3. Run `docker compose up -d`
4. Once that is finished running, navigate to `localhost:3000` and set up your pod.
5. Navigate to `localhost:5173` and check that PASS is working properly.
